### ISSUE TITLE
[Bug] Missing JSON element "microformat" (BrokenTubeException)

### ISSUE THREAD

**Issue by Ferd-fb** [iv-org/invidious#3387]
Title: `Missing JSON element "microformat" (BrokenTubeException)`
Date: `2022-11-11T18:57:11Z`
Route: `/watch?v=NhZE8qCS3NM`
Version: `2022.11.05-b267613 @ master`

<details>
<summary>Backtrace</summary>
<p>
   
```
Missing JSON element "microformat" (BrokenTubeException)
  from /usr/share/crystal/src/array.cr:114:31 in 'parse_video_info'
  from src/invidious/videos/parser.cr:92:12 in 'extract_video_info:video_id:context_screen'
  from src/invidious/videos.cr:385:18 in 'fetch_video'
  from src/invidious/videos.cr:354:13 in 'get_video:region'
  from src/invidious/routes/watch.cr:63:15 in 'handle'
  from lib/kemal/src/kemal/route.cr:13:9 in '->'
  from src/invidious/helpers/handlers.cr:30:37 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/filter_handler.cr:21:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call'
  from /usr/share/crystal/src/http/server/handler.cr:28:7 in 'call_next'
  from lib/kemal/src/kemal/init_handler.cr:12:7 in 'process'
  from /usr/share/crystal/src/http/server.cr:500:5 in '->'
  from /usr/share/crystal/src/fiber.cr:146:11 in 'run'
  from ???
```
</p>
</details>

**Comment by SamantazFox** [comment]
Does it still happen? This was probably fixed in #3238

**Comment by SamantazFox** [comment]
Seems like it was fixed, because I can't reproduce on an up to date instance.
Feel free to reopen if you thing this is still the case!
